cmd/autoheal/healer_test.go

```diff
@@ -570,9 +571,19 @@ func makeHealer(t *testing.T, name string) *Healer {
 type FakeAWXActionRunner struct{}
 
 func (f FakeAWXActionRunner) RunAction(rule *autoheal.HealingRule, action interface{}, alert *alertmanager.Alert) error {
-	glog.Infof("Fake AWXActionRunner called with rule '%s' and alert '%s'",
+	errMsg := fmt.Sprintf("Fake AWXActionRunner called with rule '%s' and alert '%s'",
 		rule.ObjectMeta.Name,
 		alert.Name(),
 	)
-	return nil
+	return errors.New(errMsg)
```

Comment:
I don't like the Fake runner returning error when everything goes well.
You can have save list of rules and alerts a specific runner was asked to run and after using it you can test the list to container what you expect.